loanie-api/controllers/thirdPartyApiControllers.js

```diff
@@ -14,7 +14,7 @@ const sendEmailNotification = (req, res) => {
 	sgMail.setApiKey(process.env.SENDGRID_API_KEY);
```

Comment:
When reading in ENV vars, it is often helpful to set a sensible default first so that your app doesn't crash if the value cannot be found.

Something like:
`const sendgrid_key = process.env.SENDGRID_API_KEY || "abc123";`
 `sgMail.setApiKey(sendgrid_key);`